content/_layouts/project.html.haml

```diff
@@ -16,12 +16,7 @@ layout: default
     articles = articles.compact.slice(0, 10)
   end
 
-:javascript
-  $(function () {
-    for (var i = 1 ; i <= 6 ; i ++) {
-      anchors.add('.container .row .col-lg9 h' + i);
```

Comment:
@oleg-nenashev there was a mising `-` here so effectively this change adds anchors to GSoC pages -- if it's not desired we can remove anchors from `project` layout.